rpcd/playbooks/roles/rpc_support/tasks/retrieve_auth_key.yml

```diff
@@ -13,6 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+- name: Install pip packages for memecache module to work
```

Comment:
Does it cause a problem (them not being installed)?  I seem to remember this coming up every so often (someone trying to add these packages back).